src/helpers/configuration_helpers.sh

```diff
@@ -110,8 +118,16 @@ function run_config_operation {
       ensure_knows_configuration
     elif [ "$operation" == "--reset" ]; then
       remove_all_configuration
+    elif [ "$operation" == "--pull-strategy" ]; then
+      if [ -z "$2" ] ; then
+        echo "usage: git town config --pull-strategy [ merge | rebase ]"
+      elif [ "$2" != 'merge' ] && [ "$2" != 'rebase' ] ; then
+        echo "The only acceptable pull strategy values are merge and rebase"
+      else 
+        store_configuration pull-branch-strategy "$2"
+      fi
```

Comment:
The main branch is dealt with by calling functions that then have logic. Are you saying you want me to put the logic into a separate file?